assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_new_edit_assignment.vm

```diff
@@ -925,6 +925,34 @@
 							</label>
 						</div>
 						#end
+						
+						<!-- Exclude Self Plag -->
+						#if($show_NEW_ASSIGNMENT_REVIEW_SERVICE_EXCLUDE_SELF_PLAG)
```

Comment:
minor comment avoid using NEW to describe this feature as it will be OLD some day!